webpack.config.js

```diff
@@ -0,0 +1,42 @@
+var path = require('path');
+var webpack = require('webpack');
+
+var config = module.exports = {
+  // the base path which will be used to resolve entry points
+  context: __dirname,
+ // the main entry point for our application's frontend JS
+  entry: './webpack/assets/javascripts/entry.js',
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: "babel-loader",
+        query: {
+          presets: ['es2015']
```

Comment:
most examples I've seen use .babelrc, do you know the difference between setting it up here vs .bablerc? (is it used elsewhere besides webpack for example).